pkg/export/setup/setup.go

```diff
@@ -50,6 +50,9 @@ const (
 	UAModeUnspecified = "unspecified"
 	UAModeAVMW        = "on-prem"
```

Comment:
Could you add comments on what each of these modes means? e.g. what's the difference between "on-prem" and "baremetal"?